### ISSUE TITLE
Standardization of energy maps

### ISSUE THREAD

**Issue by OLibykov** [WenjieZhao1/S2M#13]
Hi, I did a little research on the published code and noticed that neither the article nor the training code has any information about standardization of energy maps. As far as I understood, you feed the energy map to the 3-channel resnet50 detectron2, which broadcasts the energy map to a 3-channel image. However, detectron2 by default applies standardization with the mean and standard deviation of ImageNet. This information can be found in the directory S2M/detectron2/config/defaults.py:

Values to be used for image normalization (BGR order, since INPUT.FORMAT defaults to BGR).
To train on images of different number of channels, just set different mean & std.
Default values are the mean pixel value from ImageNet: [103.53, 116.28, 123.675]
_C.MODEL.PIXEL_MEAN = [103.530, 116.280, 123.675]
When using pre-trained models in Detectron1 or any MSRA models,
std has been absorbed into its conv1 weights, so the std needs to be set 1.
Otherwise, you can use [57.375, 57.120, 58.395] (ImageNet std)
_C.MODEL.PIXEL_STD = [1.0, 1.0, 1.0]

Have you studied this issue? Have you measured the mean and standard deviations of the PEBAL and RPL energy maps?

**Comment by WenjieZhao1** [comment]
I am so glad you did the research on this area. I think it only need several line to compute it.
